.rubocop.yml

```diff
@@ -508,7 +508,7 @@ Metrics/LineLength:
   Description: Limit lines to 100 characters.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#80-character-limits
   Enabled: true
-  Max: 100
+  Max: 80
```

Comment:
I am confused - `houndci` checks for 80 characters (at least that's what I see in my pull request comments by `houndci`: `line is to long xx/80`). In this setting file there is 100. Why it is not 100 and if it's checking for 80 shouldn't be here 80 too?